src/Microsoft.TemplateEngine.Abstractions/IFileChange.cs

```diff
@@ -12,7 +15,9 @@ public interface IFileChange
         byte[] Contents { get; }
     }
 
+#pragma warning disable CS0618 // Type or member is obsolete
     public interface IFileChange2 : IFileChange
```

Comment:
We are already introducing a lot of breaking changes. Should we merge `IFileChange2` with `IFileChange`?

Reply:
I see that this will also prevent having an `ICreationEffects2` interface and an `IGenerator.GetCreationEffects2` method.